technical-reports/color/token-naming.md

```diff
@@ -0,0 +1,393 @@
+
+#### Descriptive
+
+Descriptive names can be more emotional and human-friendly because they often relate to tangible things that people interact with, like grass or watermelon.
+
+| Pros                                                                                                   | Cons                                                                                                                          |
+| ------------------------------------------------------------------------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------- |
+| Easier for some people to identify with, could be used to distinguish brand colors from product colors | Harder to determine the scale of colors (for example: which ones are lighter vs. darker? How well do they pair together?)     |
+|                                                                                                        | Names may not be easily recognized by non-English speakers. For teams working across languages, this may not be a good choice |
+
+<aside class="example">
+
+```json
+{
+  "color": {
+    "grass": {
+      "$type": "color",
+      "$value": {
+        "colorSpace": "srgb",
+        "components": [0.67, 0.79, 0.74]
+      }
+    },
+    "brand": {
+      "watermelon": {
+        "$type": "color",
+        "$value": {
+          "colorSpace": "srgb",
+          "components": [0.91, 0.28, 0.42]
+        }
+      }
+    }
+  }
+}
+```
+
+</aside>
+
+#### Numerical
+
+##### Ordered scales
+
+Numerical tokens often follow a scale to help delineate how the colors progress. For example, when using an ordered scale, `color.blue.500` may be the base color, where the lightest color value is `color.blue.100`, and the darkest value could be `color.blue.900`, and these values are ordered in increments of 100s in between. We recommend not using sequential numbers (ex: 1, 2, 3, 4) for scalability in case future colors need to be added in between two existing colors.
+
+<aside class="example">
+
+```json
+{
+  "color": {
+    "green": {
+      "400": {
+        "$type": "color",
+        "$value": {
+          "colorSpace": "srgb",
+          "components": [0.42, 0.73, 0.63]
+        }
+      },
+      "500": {
+        "$type": "color",
+        "$value": {
+          "colorSpace": "srgb",
+          "components": [0.15, 0.56, 0.42]
+        }
+      },
+      "600": {
+        "$type": "color",
+        "$value": {
+          "colorSpace": "srgb",
+          "components": [0.07, 0.5, 0.36]
+        }
+      }
+    }
+  }
+}
+```
+
+</aside>
+
+Numerical token names can also allow for further specificity when needed. For example, when creating neutral palette tokens (like grays), the scale may increase by increments of 25 instead of 100 at the lightest values, and then increment by 100 thereafter.
+
+<aside class="example">
+
+```jsonc
```

Comment:
Should we be consistent with only using `jsonc` or `json`?

Reply:
I only used jsonc where the examples had comments; technically we could switch everything to jsonc with no meaningful difference, but imo it's probably better to stick to jsonc as the exception.